spanner/tests/unit/test_keyset.py

```diff
@@ -119,6 +131,366 @@ def test_to_pb_w_start_open_and_end_closed(self):
         self.assertEqual(len(krange_pb.end_closed), 1)
         self.assertEqual(krange_pb.end_closed.values[0].string_value, KEY_2[0])
 
+    def test_ctor_empty_list_start_closed_and_key_end_closed(self):
```

Comment:
@chemelnucfin Can you "port over" my previous comments? It seems I've seen this code 3 (?) times now and I'm having a deja vu about my comments.

Reply:
Sorry, I think in the process of breaking up the PR some old code was retained.

Reply:
Also, I think this test is different than the previous you commented.